tools/tflitefile_tool/select_operator.py

```diff
@@ -1253,6 +1254,28 @@ def GenerateModel(args, new_builder, sample_model, operator_list, new_input_tens
     return tflite.Model.ModelEnd(new_builder)
 
 
+def StoreMetadata(path, used_tensors, new_inputs, new_outputs):
+    metadata = {}
+
+    # For inputs and outputs
+    input_old_idx = []
+    input_new_idx = []
+    output_old_idx = []
+    output_new_idx = []
+    for input_tensor_idx in new_inputs:
+        input_old_idx.append(input_tensor_idx)
+        input_new_idx.append(used_tensors[input_tensor_idx])
+    for output_tensor_idx in new_outputs:
+        output_old_idx.append(output_tensor_idx)
+        output_new_idx.append(used_tensors[output_tensor_idx])
+
+    metadata["inputs"] = {"old": input_old_idx, "new": input_new_idx}
```

Comment:
(optional) (about naming)

1. I thought input_old_idx is just a number. But it is a list which could contain multiple ones. Could you help me by renaming `idx` to `indices` or `indexes`?

2. I prefer `org` to `old`. I found there are two more `old` in this source. I found it was also added by you. : )

Reply:
> 1. I thought `input_old_idx` would be just a number. But it is a list which could contain multiple ones. Could you help me by renaming `idx` to `indices` or `indexes`?

OK, I will.

> 2. I prefer `org` to `old`. I found there are two more `old` in this source. I found it was also added by you. : )

Thanks, `org` looks good.  I will change all `old` to `org`.